dask-gateway.yaml

```diff
@@ -1,13 +1,19 @@
 package:
   name: dask-gateway
   version: 2024.1.0
-  epoch: 0
+  epoch: 1
   description: "A multi-tenant server for securely deploying and managing Dask clusters."
   copyright:
     - license: BSD-3-Clause
+  options:
+    # We create a dependency on libpython even though we provide
+    # libpython in the virtual environment. This prevents python
+    # versions on the host from being swapped out. Enabling no-
+    # depends works around this
+    no-depends: true
   dependencies:
     runtime:
-      - python3
+      - python-3
```

Comment:
[non-blocking] i'd generally align to using 'python3' here, which is more consistent with the other package names. Both are an option and reference the same package

Reply:
Thank you. I'll get that switched up :)